index.js

```diff
@@ -96,8 +96,7 @@ GoogleContacts.prototype._get = function (params, cb) {
 
 GoogleContacts.prototype.getContacts = function (cb, params) {
     var self = this;
-
-    this._get(_.extend({type: 'contacts'}, params, this.params), receivedContacts);
+    this._get(_.extend({type: 'contacts'}, this.params, params), receivedContacts);
```

Comment:
line 100 of `index.js` would remove the id parameter and all others that would be defined in the function call, so a reorder was necessary.